UNRELEASED.md

```diff
@@ -2,6 +2,8 @@
 
 Use [the changelog guidelines](https://git.io/polaris-changelog-guidelines) to format new entries. 💜
 
+- Update `Tabs` to prevent `aria-valid-attr-value` a11y error ([#3777](https://github.com/Shopify/polaris-react/pull/3777))
```

Comment:
```suggestion
- Updated `Tabs` to prevent `aria-valid-attr-value` a11y error ([#3777](https://github.com/Shopify/polaris-react/pull/3777))
```

We try to use past tense